### DISCUSSION TITLE
一般要训练多少个Epoch啊，每个Epoch 花费40s时间正常吗？，我租的GPU服务器+。+

### DISCUSSION THREAD

**Discussion by faithererer** [svc-develop-team/so-vits-svc#150] [General]
一般要训练多少个Epoch啊，每个Epoch 花费40s时间正常吗？，我租的GPU服务器+。+

**Comment by hinatayiacc2** [comment]
> 一般要训练多少个Epoch啊，每个Epoch 花费40s时间正常吗？，我租的GPU服务器+。+

每个epoch的时间和你的数据量成正比这个不好说。至于训练多少个epoch，你可以自己安装tensorboard看训练曲线，觉得收敛了就训练够了。
